tensorflow/python/keras/optimizer_v2/optimizer_v2.py

```diff
@@ -910,7 +910,7 @@ def _serialize_hyperparameter(self, hyperparameter_name):
       return value()
     if tensor_util.is_tensor(value):
       return backend.get_value(value)
-    return value
+    return float(value)
```

Comment:
Not sure why this is included in this PR.

Reply:
```suggestion
    return value
```
It shouldn't be, pushed to the wrong branch. Sorry for the unnecessary delay.